pkgs/tools/admin/gixy/default.nix

```diff
@@ -11,7 +11,7 @@ python.pkgs.buildPythonApplication rec {
     owner = "yandex";
     repo = "gixy";
     rev = "v${version}";
-    sha256 = "0dg8j8pqlzdvmyfkphrizfqzggr64npb9mnm1dcwm6c3z6k2b0ii";
+    sha256 = "11aps8a8xg1nlw36jgrmnaf38imfz2rj67wnlalrrsqb616xipcv";
   };
```

Comment:
Is there a reason not to use `fetchPypi`?

Reply:
@dotlambda I did neither know about that nor think of it when I packaged this :)
Let me take a look.

Reply:
Thank's for the hint, it is also available from pypi :+1: